cli/test/fixtures/dobetterweb/dbw_tester.css

```diff
@@ -10,10 +10,10 @@ body {
 .doesnotapply {
   display: flexbox; /* FAIL */
 }
-/* LCP image in a pseudo-element */
+/* redirected LCP image in a pseudo-element */
 #toppy::before {
   content: '';
-  background-image: url('lighthouse-1024x680.jpg?lcp');
+  background-image: url('lighthouse-1024x680.jpg?lcp&redirect=lighthouse-1024x680.jpg%3Fredirected-lcp');
```

Comment:
Seems like this change should be associated with updating some artifact fixtures.

Reply:
oh right, good call